tests/components/suez_water/test_sensor.py

```diff
@@ -26,7 +26,13 @@ async def test_sensors_valid_state(
     entity_registry: er.EntityRegistry,
 ) -> None:
     """Test that suez_water sensor is loaded and in a valid state."""
-    with patch("homeassistant.components.suez_water.PLATFORMS", [Platform.SENSOR]):
+    with (
+        patch("homeassistant.components.suez_water.PLATFORMS", [Platform.SENSOR]),
+        patch(
+            "homeassistant.components.suez_water.coordinator.SuezWaterCoordinator._update_statistics",
```

Comment:
Why do we patch this?

Reply:
I did it so each platform is tested indepently from each other but beaucse the statistics did not belong to any platform I did not find another way of doing it.

But I removed this patch anyway